lib/gems/pending/appliance_console/date_time_configuration.rb

```diff
@@ -12,10 +12,10 @@ class DateTimeConfiguration
 
     include ApplianceConsole::Logging
 
-    def initialize
-      @new_date         = nil
-      @new_time         = nil
-      @manual_time_sync = false
+    def initialize(config = {})
```

Comment:
I know it's being done in other places... I just don't really like options hashes. It's to easy for it it get out of hand. Maybe kwargs?